surveys/2023-annual-survey/questions.md

```diff
@@ -94,6 +94,19 @@ Type: select one
 > those who program once a week but always in Rust and those who program daily but
 > use Rust once a week.
 
+### On average, how often do you do coding/design work in general (i.e. excluding Rust)?
```

Comment:
I think that it's both easier to answer and better (since we ask in some coarse grained way) to ask it in an inclusive sense, i.e. all coding, including Rust work.